projects/ngx-formentry/src/form-entry/value-adapters/obs-adapter-helper.ts

```diff
@@ -489,20 +489,39 @@ export class ObsAdapterHelper {
   getGroupPayload(node: NodeBase) {
     const nodeAsGroup: GroupNode = node as GroupNode;
 
-    let childrenPayload = [];
+    // Get existing obs
+    let childrenPayload =
+      nodeAsGroup?.initialValue?.groupMembers?.map((node) =>
+        this.getOldObsPayload(node)
+      ) || [];
+
+    let isGroupChanged: boolean = false;
     _.each(nodeAsGroup.children, (child) => {
       const payload = this.getObsNodePayload(child);
       if (payload.length > 0) {
-        childrenPayload = childrenPayload.concat(payload);
+        isGroupChanged = true;
+        if (payload[0].uuid) {
+          if (payload[0].voided) {
+            childrenPayload.find(
+              (obs) => obs.uuid == payload[0].uuid
+            ).voided = true;
+          } else {
+            childrenPayload.find((obs) => obs.uuid == payload[0].uuid).value =
+              payload[0].value;
+          }
+        } else {
+          childrenPayload = childrenPayload.concat(payload);
+        }
       }
     });
 
-    if (childrenPayload.length === 0) {
+    if (!isGroupChanged) {
       return null;
     }
 
     const groupPayload: any = {
```

Comment:
```suggestion
    const groupPayload = {
```

Reply:
Removing this will cause problems when I try to add attributes to the property.

Reply:
Don't worry about it then.